app/models/miq_request_workflow.rb

```diff
@@ -344,12 +344,19 @@ def rails_logger(_name, _start)
     # Rails.logger.warn("#{name} #{start.zero? ? 'start' : 'end'}")
   end
 
+  def self.handle_deprecation(text)
+    deprecated_warn = "method: parse_ws_string, arg Type => String"
+    solution = "arg should be a hash"
+    MiqAeMethodService::Deprecation.deprecation_warning(deprecated_warn, solution) if text.kind_of?(String)
+  end
```

Comment:
@syncrou If this method is just for deprecating `parse_ws_string` then I would rather have the code inline in the method.